src/backend/distributed/commands/database.c

```diff
@@ -396,6 +441,20 @@ citus_internal_database_command(PG_FUNCTION_ARGS)
 			DropDatabase(NULL, (DropdbStmt *) parseTree);
 		}
 	}
+	else if (IsA(parseTree, AlterDatabaseStmt))
+	{
+		elog(DEBUG1, "Altering DB");
+		AlterDatabaseStmt *stmt = castNode(AlterDatabaseStmt, parseTree);
+
+		bool missingOk = false;
+		Oid databaseOid = get_database_oid(stmt->dbname, missingOk);
+
+
+		if (OidIsValid(databaseOid))
+		{
+			AlterDatabase(NULL, (AlterDatabaseStmt *) parseTree, true);
```

Comment:
Out of `ALTER DATABASE` commands only `SET TABLESPACE` doesn't work inside a transaction. And there is a comment in this PR that says we don't support it now. What is the purpose of the changes in this function then?

Reply:
Removed these lines now